docs/infrastructure/installation/install-clio-on-ubuntu.md

```diff
@@ -19,7 +22,9 @@ Before you install Clio, you must meet the following requirements.
 
 - Ensure that your system meets the [system requirements](system-requirements.md).
 
-    **Note:** Clio has the same system requirements as the `rippled` server, except Clio needs less disk space to store the same amount of ledger history.
+    {% admonition type="info" name="Note" %}
+    Clio has the same system requirements as the `rippled` server, except Clio needs less disk space to store the same amount of ledger history.
+    {% /admonition %}
 
 -  You need compatible versions of CMake and Boost. Clio requires C++20 and Boost 1.75.0 or higher.
```

Comment:
Clio requires C++23 and Boost 1.83.0 or higher. More info here: https://github.com/XRPLF/clio/blob/develop/docs/build-clio.md